dotenv/godotenv_test.go

```diff
@@ -623,10 +623,10 @@ func TestSubstitutionsWithShellEnvPrecedence(t *testing.T) {
 
 	envFileName := "fixtures/substitutions.env"
 	expectedValues := map[string]string{
-		"OPTION_A": "5",
-		"OPTION_B": "5",
-		"OPTION_C": "5",
-		"OPTION_D": "55",
+		"OPTION_A": "1",
```

Comment:
Same as above - this seems incorrect now. The test is actually called `TestSubstitutionsWithShellEnvPrecedence`, but now we're asserting that env file values have higher precedence 🤔

Reply:
I also added more tests with default values to show the trick.